// containers/clothe/index.js
import React, {Component} from 'react'
import {connect} from 'react-redux'
import * as R from 'ramda'
import {Link} from 'react-router-dom'

import {fetchclotheById, addclotheToBasket} from 'actions'
import {getclotheById} from 'selectors'
import BasketCart from 'components/basketCart'

class clothe extends Component {
  componentDidMount() {
    const clotheId = this.props.match.params.id
    this.props.fetchclotheById(clotheId)
  }

  renderField() {
    const {clothe} = this.props
    const columnField = R.compose(
      R.toPairs,
      R.pick([
        'Brand',
        'Color',
        'Composition',
        'Size',
        'Gender',
        'Season',
      ])
    )(clothe)

    console.log('columns', columnField)

    return columnField.map(([key, value]) => (
      <div className='column' key={key}>
        <div className='ab-details-title'>
          <p>{key}</p>
        </div>
        <div className='ab-details-info'>
          {value}
        </div>
      </div>
    ))
  }

  renderContent() {
    const {clothe} = this.props

    return (
      <div className='thumbnail'>
        <div className='row'>
          <div className='col-md-6'>
            <img
              className='img-thumbnail'
              src={clothe.image}
              alt={clothe.name}
            />
          </div>
          <div className='col-md-6'>
            {this.renderField()}
          </div>
        </div>
        <div className='caption-full'>
          <h4 className='pull-right'>{clothe.price}$</h4>
          <h4>{clothe.name}</h4>
          <p>{clothe.description}</p>
        </div>
      </div>
    )
  }

  renderSidebar() {
    const {clothe, addclotheToBasket} = this.props

    return (
      <div>
        <p className='lead'>Quick shop</p>
        <BasketCart/>
        <div className='form-group'>
          <h1>{clothe.name}</h1>
          <h2>{clothe.price}$</h2>
        </div>
        <Link to='/' className='btn btn-info btn-block'>
          Back to store
        </Link>
        <button
          type='button'
          className='btn btn-success btn-block'
          onClick={() => addclotheToBasket(clothe.id)}
        >
          Add to cart
        </button>
      </div>
    )
  }

  render() {
    return (
      <div className='view-container'>
        <div className='container'>
          <div className='row'>
            <div className='col-md-9'>
              {this.props.clothe && this.renderContent()}
            </div>
            <div className='col-md-3'>
              {this.props.clothe && this.renderSidebar()}
            </div>
          </div>
        </div>
      </div>
    )
  }
}

const mapStateToProps = state => {
  return {
    clothe: getclotheById(state, state.clothePage.id)
  }
}

const mapDispatchToProps = {
  fetchclotheById,
  addclotheToBasket
}

export default connect(mapStateToProps, mapDispatchToProps)(clothe)
