// api/mockclothes.js
export default [
    {
        id: '1',
        categoryId: '1',
        sizeid: '48',
        name: 'Coat',
        description:
            'Winter coat. Has a high-tech insulation. Laconic cut, sitting on the figure.',
        price: 87,
        image: '/uploads/Outerwear1.jpg',
        Brand: 'Envy Lab',
        Color: 'Black',
        Composition: 'wool 65%, polyester 35%',
        Size: 'M',
        Gender: 'Male',
        Season: 'Winter',
    },
    {
        id: '2',
        categoryId: '1',
        sizeid: '46',
        name: 'Jacket',
        description:
            'This simple jacket includes a durable outer jacket made of linen-like material. And under the upper jacket, the highlight of the program is a detachable fleece jacket with a warmed and windproof front. Soft knitted fleece, effective synthetic insulation and windproof fabric are an unbeatable team for unsurpassed comfort and weather protection, even if you wear only an inner jacket. Jackets can be unfastened from each other and worn separately.',
        price: 129,
        image: '/uploads/Outerwear2.jpg',
        Brand: 'Jack Wolfskin',
        Color: 'Wine',
        Composition: 'polyamide 100%',
        Size: 'S',
        Gender: 'Female',
        Season: 'Winter',
    },
    {
        id: '3',
        categoryId: '2',
        sizeid: '50',
        name: 'Jeans',
        description:
            'Great straight-cut jeans, slightly tapered down. Five pockets. Branded Button. On the belt loops for the belt, behind the brand logo patch.',
        price: 29,
        image: '/uploads/Jeans1.jpg',
        Brand: 'Jack Wolfskin',
        Color: 'Blue',
        Composition: '100% cotton',
        Size: 'L',
        Gender: 'Male',
        Season: 'Demi-season',
    },
    {
        id: '4',
        categoryId: '3',
        sizeid: '44',
        name: 'Hoody',
        description:
            'This sweatshirt is a combination of comfort and elegant style for every day. Soft knitwear and comfortable fit will make this sweatshirt your favorite. Ideal for both the classic look and urban style.',
        price: 44,
        image: '/uploads/Sweatshirts1.jpg',
        Brand: 'PEPE JEANS',
        Color: 'Gray',
        Composition: '100% cotton',
        Ssize: 'XS',
        Gender: 'Female',
        Season: 'Winter',
    },
    {
        id: '5',
        categoryId: '3',
        sizeid: '52',
        name: 'Hoody',
        description:
            'Slim fit sweatshirt. Well sits, pleasant to the body. A high proportion of cotton provides increased wear resistance of the material.',
        price: 19,
        image: '/uploads/Sweatshirts2.jpg',
        Brand: 'Geesoo',
        Color: 'Dark grey',
        Composition: 'cotton 80%, polyester 20%',
        Size: 'XL',
        Gender: 'Male',
        Season: 'Demi-season',
    },
    {
        id: '6',
        categoryId: '4',
        sizeid: '52',
        name: 'Business suit',
        description:
            'A classic suit for every day. The upper fabric is dense, wrinkled with a small structure of heterogeneous color. The silhouette is straight to a standard or full figure. On a jacket 2 buttons - fasteners, 1 slot on the back. Classic trousers without tucks on the lining, the line of landing of the trouser belt is average, the belt is compound.',
        price: 78,
        image: '/uploads/Costumes1.jpg',
        Brand: 'STENSER',
        Color: 'Dark grey',
        Composition: 'viscose 56%, polyamide 30%, wool 9%, elastane 5%',
        Size: 'XL',
        Gender: 'Male',
        Season: 'Demi-season',
    },
    {
        id: '7',
        categoryId: '5',
        sizeid: '48',
        name: 'Sport suit',
        description:
            'Special cut sleeves of the raglan, there are no seams on the shoulder, which always get wet in the first place. The central fastener a moisture protective lightning, is closed by the valve. The hood is adjustable in the shape of the face. A belt and a bottom of sleeves on an elastic band. The bottom of the jacket is adjustable in size using the cord. The seams are glued. Two outer pockets. Two inside pockets. Pants Elastic waist. Elastic band under the legs. Two pockets on the hips, closed with flaps on the buttons.',
        price: 58,
        image: '/uploads/Sports1.jpg',
        Brand: 'SPLAV',
        Color: 'Black',
        Composition: '100% polyester',
        Size: 'M',
        Gender: 'Male',
        Season: 'Demi-season',
    },
    {
        id: '8',
        categoryId: '6',
        sizeid: '46',
        name: 'Pajamas',
        description:
            'Set of warm cotton canvas with lycra. Short-sleeved T-shirt with a rounded neckline. Mid-rise cropped trousers with elasticated waistband.',
        price: 36,
        image: '/uploads/Home_clothes1.jpg',
        Brand: 'VESTA',
        Color: 'Gray melange',
        Composition: 'cotton 95%, elastane 5%',
        Size: 'S',
        Gender: 'Female',
        Season: 'Demi-season',
    },
    {
        id: '9',
        categoryId: '2',
        sizeid: '44',
        name: 'Jeans',
        description:
            'An ideal modern model for every day. High rise and straight cropped trousers are suitable for any figure.',
        price: 27,
        image: '/uploads/Jeans2.jpg',
        Brand: 'IDGem',
        Color: 'Light blue',
        Composition: 'cotton 98%, elastane 2%',
        Size: 'XS',
        Gender: 'Female',
        Season: 'Demi-season',
    }
]
